magfest/templates/magfest_dept_checklist/allotments.html

```diff
@@ -160,25 +160,25 @@
         <br/> <br/>
         Number of cash boxes your department needs at one time:
         <input type="text" name="cash_boxes" size="3" />
-        
+
         <br/> <br/>
         For each cash box you recieve:
         <table style="width:auto">
         <tr>
             <td>Number of $1 Bills</td>
-            <td><input type="text" name="1bills" size="3" /></td>
+            <td><input type="text" name="bills_1" size="3" /></td>
```

Comment:
# NEEDS REVIEW

Jinja doesn't allow variables names to begin with a number, so I had to switch these input names around.

I think this will work, see my comment below on `allotments.txt`.